c/c++.cpp

```diff
@@ -1,9 +1,9 @@
-#include <iostream> // include API
+#include <iostream> // Include the I/O library
 
 using namespace std;
```

Comment:
Consider [removing this line](https://isocpp.org/wiki/faq/coding-standards#using-namespace-std) and put explicit `std::` qualifiers in the code.